sdk/typescript/package.json

```diff
@@ -29,7 +29,8 @@
     "test": "pnpm test:unit && pnpm test:e2e",
     "test:unit": "vitest unit",
     "test:e2e": "wait-on http://127.0.0.1:9123 -l --timeout 120000 && vitest e2e",
-    "prepare:e2e": "cargo build --bin sui-test-validator --bin sui --profile dev && RUST_LOG=info,sui=debug,anemo_tower=warn cargo run --bin sui-test-validator",
+    "test:e2enw": "vitest e2e",
+    "prepare:e2e": "cargo build --bin sui-test-validator --bin sui --profile dev && RUST_LOG=info,sui=debug,anemo_tower=warn,consensus=off cargo run --bin sui-test-validator",
```

Comment:
Cargo run should build it if it needs to right? You can probably remove the explicit build.

Reply:
This is mainly used to build the `sui` binary which is not built at the `prepare:e2e` step. The `sui` binary is needed for publishing modules and we don't want the building to happen in the middle of a test